### PR TITLE
Create pill button

### PR DESCRIPTION
**Pull request by lucianoschillagi**
### Checklist

- [x] Compiler warnings resolved
- [x] Linter warnings resolved
- [x] User-facing strings in Localizable.strings file
- [x] Unused code, print statements, and comments removed

**What this PR does**:
Create a reusable Pill Button

**Which issue(s) this PR fixes:**
#48 

**Special notes for reviewers**:
For testing purposes, please put "ButtonPrototypeController" as initial View Controller

### CODE REVIEW

File: FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class ButtonPrototypeController: UIViewController {
+
+    private var regularButton = PillButton()
```
Comment (haidalinda) [inline FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift:31]: thank you for the quick turnaround & addressing of feedback! in terms of consolidating into 1 button - i maybe miscommunicated 😅 

what we're going for is something like

```
enum PillButtonType {
    case .regular
    case .outline
    case .bold
}
```

and the ability to configure the PillButton as such, e.g. 

```
let postButton = PillButton(type: .regular, title: "PostCTA".localized)
postButton.active = false
```

File: FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  }
+
+    func setupRegularButton() {
+         view.addSubview(regularButton)
+         regularButton.setTitle("Regular Button", for: .normal)
+         regularButton.translatesAutoresizingMaskIntoConstraints = false
+         regularButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
+         regularButton.widthAnchor.constraint(equalToConstant: 280).isActive = true
+         regularButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+         regularButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -500).isActive = true
```
Comment (haidalinda) [inline FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift:52]: for constraints, we're trying to be consistent using one utility. example usage: https://github.com/FightPandemics/FightPandemics-iOS/pull/87/files#diff-5fbf4ba7b8989796d3ea7e5fd6f77722R51

File: FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift (hunk 1 above)
Comment (lucianoschillagi) [reply FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift:31]: Ok

File: FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift (hunk 2 above)
Comment (lucianoschillagi) [reply FightPandemics/Features/Shared/PillButton/ButtonPrototypeController.swift:52]: Ok. Question: Which framework does it belongs "makeSubview(of:)" to?

**Pull request closed without merging**